docs/windows.md

```diff
@@ -0,0 +1,210 @@
+# Antrea on Windows
+
+## Overview
+Antrea supports Windows worker Node. On Windows Node, Antrea sets up an overlay network to forward packets between nodes.
+Currently VXLAN and STT tunnels are supported.
+
+This page shows how to install Antrea Agent on Windows Nodes and register the Node to existing kubernetes cluster.
+
+For the detailed design of how Antrea Agent works on Windows, please refer to the doc [Antrea on Windows](
+https://docs.google.com/document/d/1lSis0XnKz8UcJSkxTgRtDhP2DAwQtcZDjT6ZRySUb48)
+
+### Components that run on Windows
+
+The following components should be configured and run on the Windows Node.
+* [kubernetes components](https://kubernetes.io/docs/setup/production-environment/windows/user-guide-windows-nodes/)
+* OVS daemons
+* Antrea Agent
+* Kube-proxy
+
+Antrea-Agent and Kube-proxy run as processes on host and are managed by management Pods. It is recommended to run OVS
+daemons as Windows services.
+
+## Deploying Antrea on Windows Worker Node
+
+### Prerequisite
+* Obtain a Windows Server 2019 license (or higher) in order to configure the
+Windows Node that hosts Windows containers. And install the latest Windows updates.
+* Deploy a Linux-based Kubernetes cluster.
+* Install [Hyper-V](https://docs.microsoft.com/en-us/windows-server/virtualization/hyper-v/get-started/install-the-hyper-v-role-on-windows-server) with management tools
+* Install [Docker](https://docs.microsoft.com/en-us/virtualization/windowscontainers/quick-start/set-up-environment?tabs=Windows-Server)
+* [Install OVS](http://docs.openvswitch.org/en/latest/intro/install/windows/)
+and configure the daemons as Windows service.
+  * The kernel driver of OVS should be [signed by Windows Hardware Dev Center](https://docs.microsoft.com/en-us/windows-hardware/drivers/install/driver-signing)
+  * If OVS driver is not signed, please refer to the Windows doc about how to [install a test-signed driver package on the test computer](https://docs.microsoft.com/en-us/windows-hardware/drivers/install/installing-a-test-signed-driver-package-on-the-test-computer).
+  * If you don't have a self-signed OVS package and just want to try the Antrea on windows, Antrea provides a
+    test-signed OVS package for you. See details in [Join Windows worker Nodes](#Join-Windows-worker-nodes) section.
+
+### Installation
+#### Download & configure Antrea for Linux
+Configure the Antrea for Linux on master Node following [Getting started](/docs/getting-started.md) document.
+```
+# Example:
+kubectl apply -f https://github.com/vmware-tanzu/antrea/releases/download/<TAG>/antrea.yml
+```
+#### Add Windows Antrea Agent and kube-proxy DaemonSets
+Now you can add Windows-compatible versions of Antrea Agent and kube-proxy by applying yaml files:
+- antrea-windows.yml
+  - Deploy Antrea Agent Windows DaemonSet.
+- kube-proxy-windows.yaml:
```

Comment:
add colon to both of the yamls, or neither of them.

Reply:
Done